commit d9c2759ebf406c5dedec7b4038137e175cd1413e (HEAD -> refs/heads/develop, refs/remotes/origin/develop, refs/remotes/origin/HEAD)
Author: Ben Webb <ben@salilab.org>
Date:   Tue Aug 18 11:38:10 2015 -0700

    Squashed 'modules/rmf/dependency/RMF/' changes from cad4f0e..50ad4d7
    
    50ad4d7 Use subprocess rather than os.system.
    
    git-subtree-dir: modules/rmf/dependency/RMF
    git-subtree-split: 50ad4d7474e51a49a4929279b3bf44457a8020e8

diff --git a/modules/rmf/dependency/RMF/tools/build/_decorators.py b/modules/rmf/dependency/RMF/tools/build/_decorators.py
index 482fa27..5f1ced8 100755
--- a/modules/rmf/dependency/RMF/tools/build/_decorators.py
+++ b/modules/rmf/dependency/RMF/tools/build/_decorators.py
@@ -2,6 +2,7 @@
 
 import os.path
 import sys
+import subprocess
 
 
 def replace(msg, to_replace):
@@ -539,10 +540,6 @@ RMF_DISABLE_WARNINGS
 
     del fl
     root = os.path.split(os.path.split(sys.argv[0])[0])[0]
-    os.system(
-        os.path.join(
-            root,
-            "dev_tools",
-            "cleanup_code.py") +
-        " " +
-        path)
+    subprocess.check_call([sys.executable,
+                           os.path.join(root, "dev_tools", "cleanup_code.py"),
+                           path])
